src/js/common/helpers/login-helpers.js

```diff
@@ -4,28 +4,13 @@ import environment from './environment.js';
 import { updateLoggedInStatus } from '../../login/actions';
 import { updateProfileField } from '../../user-profile/actions';
 
-export function handleLogin() {
-  this.serverRequest = fetch(`${environment.API_URL}/v0/sessions/new?level=1`, {
-    method: 'GET',
-  }).then(response => {
-    return response.json();
-  }).then(json => {
-    const myVerifyUrl = json.authenticate_via_get;
-    const receiver = window.open(myVerifyUrl, '_blank', 'resizable=yes,scrollbars=1,top=50,left=500,width=500,height=750');
-    receiver.focus();
-  });
-}
-
-export function handleVerify() {
-  this.serverRequest = fetch(`${environment.API_URL}/v0/sessions/new?level=3`, {
-    method: 'GET',
-  }).then(response => {
-    return response.json();
-  }).then(json => {
-    const myVerifyUrl = json.authenticate_via_get;
-    const receiver = window.open(myVerifyUrl, '_blank', 'resizable=yes,scrollbars=1,top=50,left=500,width=500,height=750');
+export function handleVerify(verifyUrl) {
+  window.dataLayer.push({ event: 'verify-link-clicked' });
+  if (verifyUrl) {
```

Comment:
Since we've now separated the actions for fetching the url and opening the window, and this `handleVerify` should only be responsible for opening the window; not sure we want to add an `else` condition that would fetch the url. I think Raquel had added a TODO to figure out a way to handle the case where the fetch of the url fails in a future PR; this probably needs more investigation before adding anything here.